src/patternfly/components/Select/select.scss

```diff
@@ -264,7 +264,7 @@
 
 .#{$select}__toggle {
   position: relative;
-  display: flex;
+  display: inline-flex;
```

Comment:
Is this needed?

Reply:
Yes, this resolves an alignment issue related to FF disliking `flex`ing buttons.

Before

![Screenshot 2023-07-07 at 7 39 13 AM](https://github.com/patternfly/patternfly/assets/5385435/31ecfcfe-2738-4bcc-b1c5-e38e42dbc914)


After

![Screenshot 2023-07-07 at 7 39 30 AM](https://github.com/patternfly/patternfly/assets/5385435/24316687-0292-44c8-b726-f334ff0f6d84)